spec/models/service_template_transformation_plan_task_spec.rb

```diff
@@ -556,9 +562,12 @@
       context 'destination is openstack' do
         let(:dst_ems) { FactoryBot.create(:ems_openstack, :api_version => 'v3', :zone => FactoryBot.create(:zone)) }
         let(:dst_cloud_tenant) { FactoryBot.create(:cloud_tenant, :name => 'fake tenant', :ext_management_system => dst_ems) }
-        let(:dst_cloud_volume_type) { FactoryBot.create(:cloud_volume_type) }
-        let(:dst_cloud_network_1) { FactoryBot.create(:cloud_network) }
-        let(:dst_cloud_network_2) { FactoryBot.create(:cloud_network) }
+        let(:dst_host) { FactoryBot.build(:host_openstack_infra, :ems_cluster => dst_cloud_tenant) }
+        # let(:dst_cloud_volume_type) { FactoryBot.create(:cloud_volume_type) }
```

Comment:
It looks like this should have been removed

Reply:
Done